### PR TITLE
Ensure UTF8 encoding

### PR DESCRIPTION
**Pull request by dgilm**
Avoid future problems with database fields encoded with latin1 and other codifications. This fixes a livestatus error (shinken-monitoring/mod-livestatus#33) creating a livestatus response with fields imported with this module encoded as latin1.

### CODE REVIEW

**Comment by Seb-Solon** [comment]
Thanks!

Same suggestion for this module about tests :)

**Comment by naparuba** [comment]
Hi,

I'm not ok with this patch. It import chardet that is not a default module. It add a new dep for a very specific case. So it should be at least optional and called only if need, and if possible avoid even with a new module parameter (like let the user set the charset if it's not utf8 for example).

File: module/module.py [hunk 1]
```diff
@@ -30,6 +30,8 @@
 except ImportError:
     MySQLdb = None
 
+import chardet
```
Comment (naparuba) [inline module/module.py:33]: PB here :)

**Comment by dgilm** [comment]
I didn't know that chardet isn't a default module although is installed in all my systems without extra actions. Sorry about that.

In that case @naparuba, I should have check if chardet is installed before using it. Check out the following patch:
https://github.com/dgilm/mod-import-mysql/commit/a36ea5aa430d2a161031a22b53f188c5768cb31d

I can prepare a new pull request if you want.

**Comment by naparuba** [comment]
Thanks :)

**Pull request merged**